### ISSUE TITLE
Name of grid_mapping variable set by CMOR

### ISSUE THREAD

**Issue by sol1105** [WCRP-CORDEX/cordex-cmip6-cmor-tables#100]
I might be mistaken, but to me it looks like one cannot influence the name of the `grid_mapping` variable when standardizing with `CMOR`. The variable seems to be hard coded to the actual name of the mapping, i.e.

```
[...]
int rotated_latitude_longitude;
[...]
float pr;
   pr: grid_mapping: "rotated_latitude_longitude";
[...]
```

This conflicts however with the [CORDEX-CMIP6 Archive Specifications](https://doi.org/10.5281/zenodo.10961069), that request `crs` as the variable name . Does it still count as `compliant to the CORDEX-CMIP6 Archive Specifications`, when having the grid_mapping variable named by CMOR or is it required to rename it to `crs`afterwards?

P.S.:
@larsbuntemeyer I see in your [CMORized test file](https://github.com/euro-cordex/py-cordex-data/blob/main/CORDEX/CMIP6/DD/EUR-12/GERICS/ERA5/evaluation/r1i1p1f1/REMO2020/v1/mon/tas/v20240529/tas_EUR-12_ERA5_evaluation_r1i1p1f1_GERICS_REMO2020_v1_mon_200001-200012.nc) that variable is labeled `rotated_latitude_longitude` as well.

**Comment by larsbuntemeyer** [comment]
Yes, thanks for pointing it out! I think it should be possible using CMOR to adapt this but right now i am also not sure if this has to be done via the grids table or keyword arguments in [cmor_set_grid_mapping](https://cmor.llnl.gov/mydoc_cmor3_api/#cmor_set_grid_mapping). For my examples, `rotated_latitude_longitude` is hardcoded [here](https://github.com/euro-cordex/py-cordex/blob/e5c32de8a6bb8512fecf0a4b020bd264b6c96668/cordex/cmor/cmor.py#L181-L187). Let me figure how to adapt the variable name of the grid mapping variable (and how we have to add it to the grids table)...

**Comment by larsbuntemeyer** [comment]
It seems like it's hardcoded after all. I'll check with PCMDI:
* https://github.com/PCMDI/cmor/issues/748
